### PR TITLE
Initialize static page

### PR DESCRIPTION
**Pull request by tboonma**

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite + React + TS</title>
```
Comment (tboonma) [inline frontend/index.html]: should be changed to sth like `Barcamp Bangkhen 2023`

**Comment by tboonma** [comment]
Please recenter this element

Screen Shot 2565-12-24 at 09 29 08

**Comment by tboonma** [comment]
Please stretch background to cover a whole page

Screen Shot 2565-12-24 at 09 30 04

Screen Shot 2565-12-24 at 09 30 27

**Review by tboonma — COMMENTED** [review]
Please consider change this

**Comment by tboonma** [comment]
Please rearrange the logo

Screen Shot 2565-12-24 at 09 34 53

File: frontend/index.html (hunk 1 above)
Comment (tboonma) [reply frontend/index.html]: Resolved in [750f15f](https://github.com/BarcampBangkhen/bcbk2023-web/pull/2/commits/750f15feea34aebc5d860d8e1196083b32d620bd)

**Comment by FreeSpirit0** [comment]
> Please recenter this element
> 
> Screen Shot 2565-12-24 at 09 29 08

Fix in 896046134d7bdb9ccdecd0db9000a29337563148

**Comment by FreeSpirit0** [comment]
> Please stretch background to cover a whole page
> 
> Screen Shot 2565-12-24 at 09 30 04
> 
> Screen Shot 2565-12-24 at 09 30 27

Fix in e7d6019cd0aba6e159f78d8f28ac9e3d7ff2395a

**Comment by redrosekana** [comment]
test

**Review by tboonma — COMMENTED** [review]
Nice, thanks

**Pull request merged**